benchmark/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.eclipse.rdf4j</groupId>
         <artifactId>rdf4j</artifactId>
-        <version>2.2-SNAPSHOT</version>
+        <version>2.3-SNAPSHOT</version>
```

Comment:
This should have been automatically done by maven. Is the benchmark module not registered as a module in the root pom?